src/scss/utilities/mixins/typography/index.scss

```diff
@@ -0,0 +1,10 @@
+//@charset 'UTF-8';
```

Comment:
charset should be at the top of each sass file.

Reply:
You're correct, my php storm was flagging all of them so i had tried commenting them out when the npm build wasn't building, sorry I need to change all of them back.